vignettes/Data_retrieval_via_API.Rmd

```diff
@@ -0,0 +1,507 @@
+
+```{r load_acid_all}
+# subset all acids by using multiple fields -- val must be same length in list form!
+acids <- tcplLoadAcid(fld = c("organism", "tissue"), 
+                      val = list("rat", "liver"),
+                      add.fld = c("aeid", "aid", "asid", "signal_direction")) 
+```
+
+```{r, echo = FALSE}
+printFormattedTable(acids)
+```
+
+### Load aid
+
+Use <font face="CMTT10">tcplLoadAid</font> to load aid and anm, as well as any other mapped fields.
+
+```{r load_aid}
+# load aid and anm for given aeid
+aid <- tcplLoadAid(fld = "aeid", val = 663)
+```
+
+```{r, echo = FALSE}
+printFormattedTable(aid)
+```
+
+Like <font face="CMTT10">tcplLoadAeid</font> and <font face="CMTT10">tcplLoadAcid</font>, <font face="CMTT10">tcplLoadAid</font> can subset with as many `fld`s as desired and can add any available additional fields.
+
+```{r load_aid_all}
+# subset all aids by using multiple fields -- val must be same length in list form!
+aids <- tcplLoadAid(fld = c("organism", "tissue"), 
+                      val = list("rat", "liver"),
+                      add.fld = c("aeid", "acid", "asid", "signal_direction")) 
+```
+
+```{r, echo = FALSE}
+printFormattedTable(aids)
+```
+
+### Load asid
+
+Use <font face="CMTT10">tcplLoadAsid</font> to load asid and asnm, as well as any other mapped fields.
+
+```{r load_asid}
+# load asid and asnm for given aeid
+asid <- tcplLoadAsid(fld = "aeid", val = 663)
+```
+
+```{r, echo = FALSE}
+printFormattedTable(asid)
+```
+
+Like the other <font face="CMTT10">tcplLoad</font> functions, <font face="CMTT10">tcplLoadAsid</font> can subset with as many `fld`s as desired and can add any available additional fields.
+
+```{r load_asid_all}
+# subset all asids by using multiple fields -- val must be same length in list form!
+asids <- tcplLoadAsid(fld = c("organism", "tissue"), 
+                      val = list("rat", "liver"),
+                      add.fld = c("aeid", "acid", "asid", "signal_direction")) 
+```
+
+```{r, echo = FALSE}
+printFormattedTable(asids)
+```
+
+### Load unit
+
+To load the normalized data type, or response unit, use <font face="CMTT10">tcplLoadUnit</font>. Notice how <font face="CMTT10">tcplLoadUnit</font> does not use `fld` and `val`, but uses `aeid` as input.
+
+```{r load_unit}
+# load resp_unit for given aeid
+unit <- tcplLoadUnit(aeid = c(663, 891))
+```
+
+```{r, echo = FALSE}
+printFormattedTable(unit)
+```
+
+## Load sample/chemical information
+
+### Load concentration unit
+
+Use <font face="CMTT10">tcplLoadConcUnit</font> to load the concentration unit for a specific `spid` or multiple `spid`s.
+
+```{r load_conc_unit}
+# load conc_unit for given spid
+conc_unit <- tcplLoadConcUnit(spid = "TP0000904H05")
+```
+
+```{r, echo = FALSE}
+printFormattedTable(conc_unit)
+```
+
+### Load chemical info
+
+Use <font face="CMTT10">tcplLoadChem</font> to load the chemical information for a specific `spid` or multiple `spid`s. Notice tcplLoadChem uses `field` instead of `fld`.
```

Comment:
Is differences in fld or field a bug? @brown-jason

Reply:
@madison-feshuk @brown-jason this is from the function docs:

The 'field' parameter is named differently from the 'fld' parameter seen in other functions because it only takes one input.